From 8f76e8e4315ab05796ec7774019c592eed3d0552 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:32 -0400
Subject: [fs] btrfs: remove unused variable from scrub_fixup_nodatasum

Message-id: <1407179109-13714-26-git-send-email-sandeen@redhat.com>
Patchwork-id: 86981
O-Subject: [RHEL7.1 PATCH 025/422] btrfs: remove unused variable from scrub_fixup_nodatasum
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit ce3e7f1073ee4958a30e5677e868f79292bc53a6
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:27 2013 +0100

    btrfs: remove unused variable from scrub_fixup_nodatasum

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/scrub.c b/fs/btrfs/scrub.c
index 1fd3f33..e5481ae 100644
--- a/fs/btrfs/scrub.c
+++ b/fs/btrfs/scrub.c
@@ -704,13 +704,11 @@ static void scrub_fixup_nodatasum(struct btrfs_work *work)
  struct scrub_fixup_nodatasum *fixup;
  struct scrub_ctx *sctx;
  struct btrfs_trans_handle *trans = NULL;
- struct btrfs_fs_info *fs_info;
  struct btrfs_path *path;
  int uncorrectable = 0;
 
  fixup = container_of(work, struct scrub_fixup_nodatasum, work);
  sctx = fixup->sctx;
- fs_info = fixup->root->fs_info;
 
  path = btrfs_alloc_path();
  if (!path) {
-- 
1.7.1